### ISSUE TITLE
Als gebruiker wil ik hulp wanneer ik mijn wachtwoord vergeten ben zodat ik de applicatie zo snel mogelijk weer kan gebruiken

### ISSUE THREAD

**Issue by peterboer** [forza-mor-rotterdam/profilr#106]
Op dit moment hebben we geen functie/content om de gebruiker te helpen als hij/zij hun wachtwoord (of gebruikersnaam) vergeten is, geblokkeerd heeft etc. Heb wellicht ff overleg met Joey/Jeoren om te zien hoe dat nu voor SBMbiel is ingericht.

**Comment by Wouterboer** [comment]
Zo te zien kun je je Rotterdam-wachtwoord wijzigen op https://wachtwoord.rotterdam.nl/

We kunnen daarnaar verwijzen. @peterboer?

**Comment by peterboer** [comment]
Tja.... Ik zie ook wel deze melding in de FAQ staan daar..
￼

Misschien ff checken bij Joey wat de flow is voor bijvoorbeeld SBMobiel?

**Comment by Wouterboer** [comment]
Joey: 

'SB-MOB is gekoppeld aan je citrix account… dus als je je wachtwoord daarvan bent vergeten dan zou je idd je wachtwoord moeten wijzigen via [wachtwoord.Rotterdam.nl](http://wachtwoord.rotterdam.nl/)'

Dit is gewoon een openbare site, wel handig voor ProfilR toch?

**Comment by peterboer** [comment]
yep helder! mooi link toevoegen aan de inlog pagina en ***@***.***+31627467159On 13 Feb 2023, at 14:09, Wouterboer ***@***.***> wrote:﻿
Joey:
'SB-MOB is gekoppeld aan je citrix account… dus als je je wachtwoord daarvan bent vergeten dan zou je idd je wachtwoord moeten wijzigen via wachtwoord.Rotterdam.nl'
Dit is gewoon een openbare site, wel handig voor ProfilR toch?
